lib/web_ui/dev/README.md

```diff
@@ -155,3 +156,22 @@ FELT_USE_SNAPSHOT=0 felt <command>
 ```
 
 _**Note**: if `FELT_USE_SNAPSHOT` is omitted or has any value other than "false" or "0", the snapshot mode will be enabled._
+
+## Upgrade Browser Version
+
+Since the engine code and infra recipes do not live in the same repository there are few steps to follow in order to upgrade a browser's version. For now these instructins are most relevant to Chrome.
+
+1. Dowload the binaries for the new browser/driver for each operaing system (macOS, linux, windows).
+2. Create CIPD packages for these packages. (More documentation is available for Googlers.)
```

Comment:
Can you provide a go/ link? Even if the docs are not publicly shared it would still be useful for when the time comes to upgrade the browser version.

Reply:
Ah I didn't know we were sharing the go links.

Reply:
done.